Sample/SampleDelegate/SampleDelegate/AppDelegate.swift

```diff
@@ -33,65 +33,65 @@ class AppDelegate: SuperDelegate, ApplicationLaunched {
         // Setup app model, networking, logging, etc.
     }
     
-    func loadInterfaceWithLaunchItem(launchItem: LaunchItem) {
+    func loadInterface(launchItem: LaunchItem) {
         // Setup our main window since we don't have a storyboard.
-        setupMainWindow(window)
+        setup(mainWindow: window)
         window.rootViewController = ViewController()
         
         switch launchItem {
-        case let .RemoteNotificationItem(remoteNotification):
+        case let .remoteNotification(item):
             // Show UI for the remote notification that brought us into the foreground.
-            didReceiveRemoteNotification(remoteNotification, notificationOrigin: .UserTappedToBringAppToForeground, fetchCompletionHandler: { (_) in
+            didReceive(remoteNotification: item, origin: .userTappedToBringAppToForeground, fetchCompletionHandler: { (_) in
                 // Nothing to do here.
             })
             
-        case let .LocalNotificationItem(localNotification):
+        case let .localNotification(item):
             // Show UI for the local notification that brought us into the foreground.
-            didReceiveLocalNotification(localNotification, notificationOrigin: .UserTappedToBringAppToForeground)
+            didReceive(localNotification: item, origin: .userTappedToBringAppToForeground)
             
-        case let .OpenURLItem(urlToOpen):
+        case let .openURL(item):
             // If we have a urlToOpen launch item, SuperDelegate has guaranteed that we can open that URL.
-            handleURLToOpen(urlToOpen)
+            let _ = handle(urlToOpen: item)
             
-        case let .ShortcutItem(shortcutItem):
+        case let .shortcut(item):
             // Process the shortcut item.
-            handleShortcutItem(shortcutItem, completionHandler: {
+            handle(shortcutItem: item, completionHandler: {
                 // Nothing to do here.
             })
             
-        case let .UserActivityItem(userActivity):
+        case let .userActivity(item):
             // If we have a userActivity launch item, SuperDelegate has guaranteed that we can continue this user activity.
-            continueUserActivity(userActivity, restorationHandler: { (_) in
+            let _ = resume(userActivity: item, restorationHandler: { (_) in
                 // Nothing to do here.
             })
             
-        case .NoItem:
+        case .none:
```

Comment:
I don't think we can optimize for not conflicting with other enum values... prefixing is gone, search is hard, and I think we kinda just have to deal with it.

Crossing my fingers for compiler-assisted search coming in Xcode 9.